packages/elements/src/components/ui/buttons/NewButton.module.css

```diff
@@ -0,0 +1,118 @@
+
+  --swui-button-background-color-hover: #1d64ab;
+  --swui-button-border-color-hover: var(--swui-button-border-color);
+  --swui-button-text-color-hover: var(--swui-button-text-color);
+  --swui-button-box-shadow-hover: 0 1px 1px 1px rgba(29, 100, 171, 0.16),
+    0 0 1px 0 rgba(29, 100, 171, 0.16);
+
+  --swui-button-background-color-active: #143e62;
+  --swui-button-border-color-active: var(--swui-button-border-color);
+  --swui-button-text-color-active: var(--swui-button-text-color);
+  --swui-button-box-shadow-active: none;
+
+  --swui-button-background-color-disabled: #e4e5e9;
+  --swui-button-border-color-disabled: #ced0d7;
+  --swui-button-text-color-disabled: #989cab;
+  --swui-button-box-shadow-disabled: none;
+
+  margin: 8px;
+  padding: var(--swui-button-padding-vertical)
+    var(--swui-button-padding-horizontal);
+  border: 1px solid var(--current-border-color, var(--swui-button-border-color));
+  background-color: var(
+    --current-background-color,
+    var(--swui-button-background-color)
+  );
+  border-radius: var(--swui-button-border-radius);
+  box-shadow: var(--current-box-shadow, var(--swui-button-box-shadow));
+  color: var(--current-text-color, var(--swui-button-text-color));
+  font-size: var(--current-text-size, var(--swui-button-text-size));
+  line-height: var(--current-size, var(--swui-button-size));
+  font-weight: 400;
+  font-family: var(--swui-font-buttons);
+  cursor: pointer;
+  user-select: none;
+  display: flex;
+  align-items: center;
+
+  transition: all var(--swui-animation-time-medium);
+
+  &:hover {
+    --current-background-color: var(--swui-button-background-color-hover);
+    --current-border-color: var(--swui-button-border-color-hover);
+    --current-text-color: var(--swui-button-text-color-hover);
+    --current-box-shadow: var(--swui-button-box-shadow-hover);
+  }
+
+  &:focus {
+    outline: 0;
+    --current-background-color: var(--swui-button-background-color-focus);
+    --current-border-color: var(--swui-button-border-color-focus);
+    --current-text-color: var(--swui-button-text-color-focus);
+    --current-box-shadow: var(--swui-button-box-shadow-focus);
+  }
+
+  &:active {
+    --current-background-color: var(--swui-button-background-color-active);
+    --current-border-color: var(--swui-button-border-color-active);
+    --current-text-color: var(--swui-button-text-color-active);
+    --current-box-shadow: var(--swui-button-box-shadow-active);
+  }
+
+  &:disabled {
+    --current-background-color: var(--swui-button-background-color-disabled);
+    --current-border-color: var(--swui-button-border-color-disabled);
+    --current-text-color: var(--swui-button-text-color-disabled);
+    --current-box-shadow: var(--swui-button-box-shadow-disabled);
+    cursor: not-allowed;
+  }
+
+  &.iconButton {
+    padding: 8px;
+    color: var(--current-text-color, var(--swui-button-text-color));
+  }
+
+  .iconLeft,
+  .iconRight {
+    height: var(--current-size, var(--swui-button-size));
```

Comment:
Set only height, no width is needed.
And make it a bit smaller on the large button, 20px looks good (Ozcar approved).